src/accelerate/state.py

```diff
@@ -34,6 +35,20 @@
 except ImportError:
     _tpu_available = False
 
+try:
+    import deepspeed
+
+    _deepspeed_available = True
+except:
+    _deepspeed_available = False
+
+try:
+    from apex import amp
+
+    _apex_available = True
+except ImportError:
+    _apex_available = False
+
```

Comment:
Apex is not supported yet. Where do you need it?

Reply:
its needed in DeepSpeedEngineWrapper.

Reply:
Ok, then please use the same as below and not a direct import.